bigquery/tests/system.py

```diff
@@ -826,6 +826,43 @@ def test_load_table_from_dataframe_w_explicit_schema(self):
         self.assertEqual(tuple(table.schema), table_schema)
         self.assertEqual(table.num_rows, 3)
 
+    def test_load_table_from_json_basic_use(self):
```

Comment:
Can we also add a system test with an autodetected schema?

Reply:
Yes. We. Can. :)